subprojects/gst-plugins-bad/sys/msdk/gstmsdkav1dec.c

```diff
@@ -70,11 +71,19 @@ static GstStaticPadTemplate sink_factory = GST_STATIC_PAD_TEMPLATE ("sink",
     GST_STATIC_CAPS ("video/x-av1")
     );
 
+#ifndef _WIN32
 static GstStaticPadTemplate src_factory = GST_STATIC_PAD_TEMPLATE ("src",
     GST_PAD_SRC,
     GST_PAD_ALWAYS,
     GST_STATIC_CAPS (GST_MSDK_CAPS_STR (COMMON_FORMAT, COMMON_FORMAT)
         VA_SRC_CAPS_STR));
+#else
+static GstStaticPadTemplate src_factory = GST_STATIC_PAD_TEMPLATE ("src",
+    GST_PAD_SRC,
+    GST_PAD_ALWAYS,
+    GST_STATIC_CAPS (GST_MSDK_CAPS_STR (COMMON_FORMAT, COMMON_FORMAT)
+        D3D11_SRC_CAPS_STR));
+#endif
```

Comment:
Since Yinhang will add dynamic query caps very sooner and this part of code will be removed. And the merged msdkvpp and msdkenc are all like this current style. Can we just keep this style for now?